python/orca/src/bigdl/orca/learn/tf2/ray_estimator.py

```diff
@@ -481,15 +482,13 @@ def predict(self, data, batch_size=None, verbose=1,
 
         return result
 
-    def get_model(self, sample_input=None):
+    def get_model(self):
```

Comment:
I think we need to keep this API in case users want to get the model to driver?
Just get the model of the first worker and return?

Reply:
only get model states like weights? Since ray worker cannot deserialize a tf model.

Reply:
oh really? Then just rename to `get_weights`?

Reply:
Or do you think we need to keep this API?

Reply:
I don't think we need this API, but `get_model` is a static method in `OrcaRayEstimator`. I prefer we rewrite this API as getting model states?

Reply:
Is it possible to do like this:
- If model_creator is defined, do as what we did previously
- If model_creator is not specified, we fetch model load path to local and load + get states from the first worker + write the states to loaded model?